.circleci/config.yml

```diff
@@ -16,6 +18,13 @@ jobs:
 
     steps:
       - checkout
+      - jq/install
+      - aws-cli/install
+      - run:
+          name: Source PyPi creds
+          command: |
+            echo "export PYPI_PASSWORD=$(aws secretsmanager get-secret-value --secret-id admin/cicd/circleci/context/pypi | jq -r '.SecretString | fromjson.password')" >> $BASH_ENV
```

Comment:
Apropos of... well, not this, but it would be nice if users could retrieve those username/password credentials for manual deployments on occasion (I know, getting everything's deployment onto CI would be preferable so this may or may not be worth it)

Reply:
Checking out PyPi the other day, it looks like we can invite other maintainers to publish to PyPi. I'd prefer setting that up over sharing creds to this account.

Overall, though, I'm not sure why we'd want to allow for manual deployments. Triggering publications through Github releases encourages review and more accountable process.